src/modules/main/Main.tsx

```diff
@@ -70,6 +71,7 @@ const Main = ({ match }: RouteChildrenProps) => {
 
 export default (props: RouteChildrenProps) => (
   <NotificationsProvider>
+    <GithubConnect />
```

Comment:
Add this inside Main just after Sidebar - using this in this place tells me GithubConnect needs Provider data to work